src/ResourceManager/Resources/Commands.Resources/ActiveDirectory/NewAzureADApplicationCommand.cs

```diff
@@ -0,0 +1,181 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ActiveDirectory.Models;
+using Microsoft.Azure.Commands.Resources.Models.ActiveDirectory;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System;
+
+namespace Microsoft.Azure.Commands.ActiveDirectory
+{
+    /// <summary>
+    /// Creates a new AD application.
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "AzureADApplication", DefaultParameterSetName = ParameterSet.ApplicationWithoutCredential), OutputType(typeof(PSADApplication))]
+    public class NewAzureADApplicationCommand : ActiveDirectoryBaseCmdlet
+    {
+        private string _KeyType = "AsymmetricX509Cert";
+        private string _KeyUsage = "Verify";
```

Comment:
You can remove these variables totally and set the properties to the default value in a constructor.